### DISCUSSION TITLE
How to compile the Tomb1Main.exe?

### DISCUSSION THREAD

**Discussion by byblo** [LostArtefacts/TRX#656] [Q&A] [answered]
Is there a guide around?

Been trying for 3h but all the tools I managed to use like VS2022 or meson does not seems to help.

**Comment by oziphantom** [comment, marked as answer]
it's in the https://github.com/rr-/Tomb1Main/blob/develop/CONTRIBUTING.md yeah I know ;) Basically it needs a posix system, WSL, MSYS, CYGWIN et al and then you run docker to do the builds.

**Comment by byblo** [comment]
Looks like it will be a long way, but it worth it :)

EDIT: Been a long fight with the Docker app which complained about a lot of things but eventually it compiled.
